setup.cfg

```diff
@@ -2,3 +2,11 @@
 ignore =
     *.cfg
     bootstrap.py
+
+[isort]
+force_alphabetical_sort = True
+force_single_line = True
+lines_after_imports = 2
+line_length = 200
+not_skip = __init__.py
+
```

Comment:
@jensens that's the plan for isort 5  but I have no idea how close we are to that, by now what I do is follow these guidelines: http://jenkinsploneorg.readthedocs.io/en/latest/run-qa-on-package.html

So in short, always do a 
```
wget https://raw.githubusercontent.com/plone/plone.recipe.codeanalysis/master/.isort.cfg -O .isort.cfg
```

before running bin/code-analysis

We could include those settings directly on p.r.codeanalysis, but then we make that a Plone-only thingie.